### ISSUE TITLE
 关于Flash的快速问题 (Quick Question About Flash)

### ISSUE THREAD

**Issue by Bushido89** [bilibili/flv.js#527]
尊敬的先生/女士，

首先，很抱歉，如果此Google翻译不合格-原始英文文本可以在下面找到。

我只是（希望）有一个简短的问题，想知道您能否提供帮助。

第三方要求我使用flv.js来支持与其API和我们的Web应用程序之一的集成。

这种集成的最终用户坚称他们不希望使用Flash，而是希望使用替代方案。

据我所知，该库使用Flash Live Video文件格式，但是不需要Flash Player。

那么说这个库不使用Flash Player，而是使用Flash文件格式，是否正确呢？

如果这是正确的，在我看来该库确实使用Flash-但我很好奇您的想法。

亲切的问候，
斯图尔特

Dear Sir/Madam,

First of all, apologies if this Google translation is substandard - the original English text can be found below.

I just had (hopefully) a quick question and wondered if you could help.

I am being asked by a 3rd party to use flv.js to support an integration with their API and one of our web applications.

The end user of this integration is adamant that they do not wish to use Flash and would prefer an alternative.

From what I can see, this library uses the Flash Live Video file format, but removes the need for the Flash Player.

Is it then a correct statement to say that this library does not use Flash Player, but does use a Flash file format?

If this is correct, it would seem to me that the library does use Flash - but I am curious to hear your thoughts.

Kind Regards,
Stewart

**Comment by winshining** [comment]
No. The library doesn't use Flash Player, it only transforms flv files or streams to mp4 segments (fmp4) and feeds them into `<video>` element via [MSE](https://www.w3.org/TR/media-source) APIs, then browsers can play the segments (Safari sucks 'cause it doesn't support MSE).
